CONTRIBUTING.md

```diff
@@ -176,6 +180,18 @@ against a variety of Operating Systems using Vagrant and the scripts in
 [pbTestScripts](https://github.com/AdoptOpenJDK/openjdk-infrastructure/tree/master/ansible/pbTestScripts)
 to validate them.
 
+## Dockerfiles as base images for our builds
```

Comment:
We should document when/how these are created/pushed to dockerhub

Reply:
@gdams Any insight into that?

Reply:
updated

Reply:
I don't feel that this sentence provides enough info about the process.
>When a change lands into master, the relevant dockerfiles are built using their relevant CI systems and pushed up to Dockerhub where they can be consumed by our jenkins build agents.

If someone at Adopt had to take over this process I feel they would be at a disadvantage not having links to build files or ci jobs. It basically says "it's automated".

Reply:
AFAICT they would need to have access to Travis and GitHub Actions for which the source code is linked in the table below.  I'm likely missing something here though :-)